### PR TITLE
Milestone 3/4

### PR DESCRIPTION
**Pull request by paulzay**
Create calculate.js
Create operate.js
Import calculate in app.js

### CODE REVIEW

**Comment by paulzay** [comment]
Kindly approve milestone3 as well

File: src/components/App.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import '../App.css';
 import ButtonDisplay from './ButtonDisplay';
 import Display from './Display';
+// import calculate from '../logic/calculate';
```
Comment (MuriloRoque) [inline src/components/App.js]: - [ ] Please uncomment this line of code.

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  } else if (operation === 'x') {
+    result = a.times(b);
+  } else if (operation === '÷') {
+    if (numberTwo !== '0') {
+      result = a.div(b);
+    } else {
+      return 'Error';
+    }
+  } else if (operation === '%') {
+    result = a.times(0.01);
```
Comment (MuriloRoque) [inline src/logic/operate.js]: - [ ] Instead of using `times` here, try using the `modifier` function to turn a number into a percentage 😉

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
### Status: Changes requested 🔴 

### Hello @paulzay 👋 

### You are almost there, your logic is great, we just need to fix some minor issues, please check my comments in your code below 😄 

Kindly make the requested changes and submit for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by Forison — APPROVED** [review]
## Status::Approved✔️
# 🙋‍♂️ @paulzay 
### You have done a good job on this one, the project meets the requirement hence approved.
### Kindly merge the branch and proceed to the next task.
##### Happy coding 👨‍💻 
Addo Forison

**Pull request merged**